src/BitNode/ui/BitNodeAdvancedOptions.tsx

```diff
@@ -342,6 +372,44 @@ export function BitNodeAdvancedOptions({
             text="Disable Sleeves' experience and augmentation"
             tooltip="Sleeves cannot gain experience or install augmentations"
           />
+          <OptionSwitch
+            disabled={Player.skills.intelligence <= 0}
+            checked={false}
+            onChange={(value) => {
+              if (!value) {
+                setIntelligenceOverride("");
```

Comment:
UI-wise, this is a bit awkward: It feels strange for the value in the input to be changed when the option is toggled, rather than just being disabled. 

I think you want the input to start with skills.intelligence as the default value (and be disabled), but it can't be coupled so tightly to the value of the BitnodeOption.